app/install/debian.md

```diff
@@ -11,15 +11,16 @@ breadcrumbs:
 
 Start by downloading the corresponding package for your configuration:
 
-- [7 Wheezy]({{ site.links.download }}/{{site.data.kong_latest.version}}/kong-{{site.data.kong_latest.version}}.wheezy_all.deb)
-- [8 Jessie]({{ site.links.download }}/{{site.data.kong_latest.version}}/kong-{{site.data.kong_latest.version}}.jessie_all.deb)
+- [7 Wheezy]({{ site.links.download }}/kong-community-edition-deb/download_file?file_path=dists/kong-community-edition-{{site.data.kong_latest.version}}.wheezy_all.deb)
+- [8 Jessie]({{ site.links.download }}/kong-community-edition-deb/download_file?file_path=dists/kong-community-edition-{{site.data.kong_latest.version}}.jessie_all.deb)
 
 ### APT Repositories
 
-You can also install Kong by using the following APT repositories and following the Bintray instructions:
+You can also install Kong via APT; follow the instructions on the "Set Me Up"
+section on the page below, setting  *distribution* to the appropriate value
+(e.g., `wheezy`) and *components* to `main`.
```

Comment:
This is a controversial topic; some say that it depends. I prefer to go with what's intuitive; "e.g." means "for instance" and a comma should follow in that case. Same goes for "i.e.", which would translate to something like "that is". After a quick Google search, I'd prefer to keep the commas.